pom.xml

```diff
@@ -49,6 +49,7 @@
   </modules>
 
   <properties>
+    <!-- INFO: When changing the Plugin API version here, also update it in the Plugin API repo README! -->
```

Comment:
I think the comment can help to remind us to change the version, but it cannot be changed right away, as the core needs to be released and used by a released version of the IDE before it can be actually changed in the README. Maybe a better place to put a reminder would be the IDE release process?